src/main/webapp/assets/general.css

```diff
@@ -0,0 +1,50 @@
+label {
```

Comment:
It might make more sense if this file were called "common.css" (especially with the changes mentioned above)